slo_generator/exporters/prometheus_self.py

```diff
@@ -27,8 +28,8 @@
 class PrometheusSelfExporter(MetricsExporter):
     """Prometheus exporter class which uses
     the API mode of itself to export the metrics."""
-    REGISTERED_URL = False
-    REGISTERED_METRICS = {}
+    REGISTERED_URL: bool = False
+    REGISTERED_METRICS: Dict = {}
```

Comment:
`dict` (with a lowercase `d` at the beginning) is probably enough here as we do not specify the key/value types. This works in Python 3.6 and 3.7 too (as we need to support these versions). Then you can remove the `typing` import on line 19.